yarn.lock

```diff
@@ -1993,10 +1993,10 @@
     "@typescript-eslint/types" "4.4.0"
     eslint-visitor-keys "^2.0.0"
 
-"@uniswap/token-lists@^1.0.0-beta.15":
-  version "1.0.0-beta.15"
-  resolved "https://registry.yarnpkg.com/@uniswap/token-lists/-/token-lists-1.0.0-beta.15.tgz#93d2e9b2b2d2cfb0ce7fb4d30fffb685efde4256"
-  integrity sha512-cxubml6fPCjPPr8tWnA3IQkYbTocsZTJWac2C5WRXoHiSd7O4xjuBISdGLT9dW7IojKYpMXnCPJS8kAwqqYXDQ==
+"@uniswap/token-lists@^1.0.0-beta.19":
```

Comment:
do you need our token list here?

Reply:
That package is more of a utility for working with tokenlists
The description below is from the repo
`This package includes a JSON schema for token lists, and TypeScript utilities for working with token lists.`